### ISSUE TITLE
[ossia-pd] add support for sending bundle

### ISSUE THREAD

**Issue by avilleret** [OSSIA/libossia#112]
but how to handle that ?
should the user choose what to put in a bundle ?
or should it be automatically ? for example, each message in one tick goes to one bundle ?

**Comment by bltzr** [comment]
maybe we could choose the interval between which to put into a bundle
like 5 or 15 ms
would that make sense ?
what if several messages are sent to the same address in this interval ? does it keep the last one only ?

**Comment by jcelerier** [comment]
> would that make sense ?

yes, but it would change an *awful* lot of stuff in the implementation because now you have to have a scheduler and you can't send stuff at different rhythms anymore

**Comment by bltzr** [comment]
we're talking pd here, not i-score, right ?

what other way to handle this could there be ?

**Comment by jcelerier** [comment]
> we're talking pd here, not i-score, right ?

sure, but to make sense it would have to be at the libossia level, not the ossia-{pd,max,...} level because it makes senses in all the environments; and I don't see how to implement this without having to change a lot of code in each environment or enforcing this behaviour also in i-score
